protostar/scripting/deployments/invoke.py

```diff
@@ -0,0 +1,14 @@
+from typing import Optional
+
+from protostar.scripting.typing import CairoOrPythonData, Fee
+
+
+def invoke(
```

Comment:
I wonder if we shouldn't have separate function for deploy to make it simpler to user, even though it is invoke under the hood

Reply:
Yes, I agree. But I don't know what signature would make sense yet and is more kind of addition, not necessity initially.